From f43c067ccefcf86638404009ad86c79ef5731269 Mon Sep 17 00:00:00 2001
Message-ID: <f43c067ccefcf86638404009ad86c79ef5731269.1745564207.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1745564207.git.sharlatanus@gmail.com>
References: <cover.1745564207.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 25 Apr 2025 07:55:47 +0100
Subject: [PATCH 1/2] gnu: Remove python-jupyter-kernel-mgmt.

* gnu/packages/jupyter.scm (python-jupyter-kernel-mgmt): Delete variable.

Change-Id: I26660a1e4f32f7e84e253493973bd1c558b98e10
---
 gnu/packages/jupyter.scm | 45 ----------------------------------------
 1 file changed, 45 deletions(-)

diff --git a/gnu/packages/jupyter.scm b/gnu/packages/jupyter.scm
index 5c95d6a144..9819caa40b 100644
--- a/gnu/packages/jupyter.scm
+++ b/gnu/packages/jupyter.scm
@@ -190,51 +190,6 @@ (define-public python-jupyter-protocol
     (license license:bsd-3)
     (properties '((upstream-name . "jupyter_protocol")))))
 
-(define-public python-jupyter-kernel-mgmt
-  (package
-    (name "python-jupyter-kernel-mgmt")
-    (version "0.5.1")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "jupyter_kernel_mgmt" version))
-              (sha256
-               (base32
-                "0977ixfi1pzjgy84hl0zycg4wpllmid98fhzcpy0lxd322w4sl7x"))))
-    (build-system python-build-system)
-    (arguments
-     (list
-      ;; There are 8 test failures, most of them in 'test_client_loop.py'
-      ;; (see: https://github.com/takluyver/jupyter_kernel_mgmt/issues/48).
-      #:tests? #f
-      #:phases
-      #~(modify-phases %standard-phases
-          (replace 'check
-            (lambda* (#:key tests? #:allow-other-keys)
-              (when tests?
-                (setenv "HOME" "/tmp")
-                (invoke "pytest" "-vv" "jupyter_kernel_mgmt")))))))
-    (propagated-inputs
-     (list python-dateutil
-           python-entrypoints
-           python-jupyter-core
-           python-jupyter-protocol
-           python-pyzmq
-           python-tornado-6
-           python-traitlets))
-    (native-inputs
-     (list python-async-generator
-           python-ipykernel
-           python-ipython
-           python-pytest
-           python-pytest-asyncio))
-    (home-page "https://jupyter.org")
-    (synopsis "Discover, launch, and communicate with Jupyter kernels")
-    (description
-     "This package is an experimental refactoring of the machinery for
-launching and using Jupyter kernels.")
-    (license license:bsd-3)
-    (properties '((upstream-name . "jupyter_kernel_mgmt")))))
-
 (define-public python-jupyter-kernel-test
   (package
     (name "python-jupyter-kernel-test")
-- 
2.49.0

